conf.pri.windows

```diff
@@ -13,17 +13,18 @@ LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
 # Adapt the lib names/versions accordingly
 # If you want to use Boost auto-linking then disable
 # BOOST_ALL_NO_LIB below and omit Boost libraries here
+LIBS += zlib.lib libcrypto.lib libssl.lib
 CONFIG(debug, debug|release) {
     LIBS += libtorrentd.lib \
-            libboost_system-vc140-mt-sgd-1_64.lib
+            boost_system-vc140-mt-d.lib
 }
 else {
     LIBS += libtorrent.lib \
-            libboost_system-vc140-mt-s-1_64.lib
+            boost_system-vc140-mt.lib
 }
-LIBS += libeay32.lib ssleay32.lib
-LIBS += zlib.lib
+
 # ...or if you use MinGW
+#LIBS += libz libcrypto libssl
```

Comment:
this too.

Reply:
@Chocobo1
I noticed both branches of the conditional of the mingw config are the same. So is the suffix the same in both configurations? or are the debug suffixes missing? Please let me know so I can fix it.

Reply:
I won't always have answer to your questions, it would be better to make the change *after* you found an answer.

Reply:
@Chocobo1 I'm only asking you because I figured you would probably know, since you have used mingw in other contexts: https://github.com/Chocobo1/qbittorent_msys2. I personally don't use it and don't really care about it, so I'd rather not have to test it myself if others already know the answer.